test/protractor.conf.js

```diff
@@ -21,7 +25,7 @@ const screenshotReporter = new HtmlScreenshotReporter({
 const junitReporter = new jasmineReporters.JUnitXmlReporter({
   consolidateAll: true,
   savePath: `${output}/junit`,
-  filePrefix: 'e2e-results'
+  filePrefix: `${timeStamp}-e2e-results`
```

Comment:
Here I wasnt really sure if we want to keep only the last test result, or we wanna all of them, so I've added timestamp to the filename. Although now I think it would be better to just keep the latest and since we dont intend to store the results anywhere. Instead we could set the result scraping after the tests are done ?
Thoughts ?

Reply:
Yeah, I've been thinking about that too.  Def looking forward to spending more time fiddling with Prometheus itself & seeing how these metrics get aggregated.  

We could always use an environment var to timestamp them, but default to not using the timestamp.  How do you feel about that?

Reply:
yeah so the results are relatively small ATM but not sure what we end up with. Safe way would be to store just the latest and set the cron to run the test every new hour and schedule the scrapping after that.

Reply:
Makes sense to me!